apps/emqx_ft/src/emqx_ft_schema.erl

```diff
@@ -282,6 +291,15 @@ schema(filemeta) ->
             {segments_ttl, hoconsc:mk(pos_integer())},
             {user_data, hoconsc:mk(json_value())}
         ]
+    };
+schema(command_response) ->
+    #{
+        roots => [
+            {vsn, hoconsc:mk(string(), #{default => <<"0.1.0">>})},
+            {topic, hoconsc:mk(string())},
+            {packet_id, hoconsc:mk(pos_integer())},
+            {result_code, hoconsc:mk(non_neg_integer())}
```

Comment:
Given that we have response schema, it now seems natural to provide richer, file-transfer-specific feedback, and not just MQTT Reason Code. The latter reduces a lot of different situations to single _unspecified error_.

Admittedly, it needs some design work, so maybe this is followup PR's material.

Reply:
Yes, I planned to implement this after the PoC, done now.